lib/exceptions.h

```diff
@@ -85,7 +85,7 @@ class MYSQLPP_EXPORT Exception : public std::exception
 
 	/// \brief Create exception object
 #if __cplusplus >= 201103L
-	Exception(std::string w) throw() :
+	Exception(std::string w) noexcept :
```

Comment:
This is not a "Tabulation typo fix", it's a change in method signature! If I get the sense from you that I can't trust what the commit contains based solely on the comment, I will eventually block you as a committer.

Reply:
I apologize. In the future, I will break down changes more consciously.